docs/operations/images.md

```diff
@@ -1,167 +1,237 @@
 
-## CentOS
+```yaml
+additionalUserData:
+  - name: busterfix.sh
+    type: text/x-shellscript
+    content: |
+      #!/bin/sh
+      update-alternatives --set iptables /usr/sbin/iptables-legacy
+      update-alternatives --set ip6tables /usr/sbin/ip6tables-legacy
+      update-alternatives --set arptables /usr/sbin/arptables-legacy
+      update-alternatives --set ebtables /usr/sbin/ebtables-legacy
+```
 
-CentOS7 support is still experimental, but should work. Please report any issues.
+Available images can be listed using:
 
-The following steps are known:
+```bash
+aws ec2 describe-images --region us-east-1 --output table \
+  --owners 136693071363 \
+  --query "sort_by(Images, &CreationDate)[*].[CreationDate,Name,ImageId]" \
+  --filters "Name=name,Values=debian-10-amd64-*"
+```
 
-* You must accept the agreement at http://aws.amazon.com/marketplace/pp?sku=aw0evgkw8e5c1q413zgy5pjce
-* Specify the AMI by id (there are no tags): us-east-1: `ami-01ed306a12b7d1c96`
-* You may find public images from the [CentOS AWS page](https://wiki.centos.org/Cloud/AWS) but the table may not be up to date.
-* You can also query by product-code: `aws ec2 describe-images --region=us-west-2 --filters Name=product-code,Values=aw0evgkw8e5c1q413zgy5pjce Name=architecture,Values=x86_64 'Name=name,Values=CentOS*' --query 'sort_by(Images,&Name)'`
+## Ubuntu 18.04 (Bionic)
 
-Be aware of the following limitations:
+Ubuntu 18.04 is fully supported and functional.
 
-* CentOS 7.2 is the recommended minimum version
-* CentOS7 AMIs are running an older kernel than we prefer to run elsewhere
+It is based on Kernel version **4.15** which has number of known bugs that affect it and may be noticed with larger clusters:
 
+* https://github.com/kubernetes/kubernetes/issues/56903
+* https://github.com/kubernetes/kubernetes/issues/67577
 
-## RHEL7
+Available images can be listed using:
 
-RHEL7 support is still experimental, but should work. Please report any issues.
+```bash
+aws ec2 describe-images --region us-east-1 --output table \
+  --owners 099720109477 \
+  --query "sort_by(Images, &CreationDate)[*].[CreationDate,Name,ImageId]" \
+  --filters "Name=name,Values=ubuntu/images/hvm-ssd/ubuntu-bionic-18.04-amd64-*"
+```
 
-The following steps are known:
+## Ubuntu 20.04 (Focal)
 
-* Redhat AMIs can be found using `aws ec2 describe-images --region=us-east-1 --owner=309956199498 --filters Name=virtualization-type,Values=hvm 'Name=name,Values=RHEL-*GA*' Name=architecture,Values=x86_64 --query 'sort_by(Images,&Name)'`
-* You can specify the name using the `redhat.com` owner alias, for example `redhat.com/RHEL-7.2_HVM-20161025-x86_64-1-Hourly2-GP2`
+Ubuntu 20.04 support was introduced in **Kops 1.16** and is fully functional. 
```

Comment:
@hakman In #8954 you say Ubuntu 20.04 support was added Kops 1.16.2, but this says it was added in 1.16.0. Which is it?

Reply:
Was cherrypicker a bit lately. Probably the release notes has to be updated for 1..16.2. Thanks!